From 8d90671eddf222296fbf81b7fbebae6ae76387bd Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin2.jansa@lgepartner.com>
Date: Tue, 11 Feb 2025 10:18:16 +0000
Subject: [PATCH] makefile_build/Makefile: drop -fuse-ld=gold

:Release Notes:
gold is marked as deprecated in GNU Binutils 2.44 and will be removed in future release:
https://lists.gnu.org/archive/html/info-gnu/2025-02/msg00001.html
https://lists.openembedded.org/g/openembedded-core/message/210627
https://lists.openembedded.org/g/openembedded-core/message/210940

:Detailed Notes:
Support for gold was removed from oe-core in:
https://git.openembedded.org/openembedded-core/commit/?id=a4addb9ab63011e7c604fc5daff95559e7d214e7

and this is now failing with:
http://gecko.lge.com:8000/Errors/Details/1109731

Building libedgetpu.so
collect2: fatal error: cannot find 'ld'
compilation terminated.
make: *** [Makefile:201: libedgetpu] Error 1

:Testing Performed:
Only build tested.

:QA Notes:
No change to image.

:Issues Addressed:
[WRR-5443] Create GPVB with Yocto 5.2 Walnascar
---
Upstream-Status: Pending

 makefile_build/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/makefile_build/Makefile b/makefile_build/Makefile
index e613e64..9044b93 100644
--- a/makefile_build/Makefile
+++ b/makefile_build/Makefile
@@ -29,7 +29,6 @@ LIBEDGETPU_LDFLAGS := \
 	-shared \
 	-Wl,--soname,libedgetpu.so.1 \
 	-Wl,--version-script=$(BUILDROOT)/tflite/public/libedgetpu.lds \
-	-fuse-ld=gold \
 	-lflatbuffers \
 	-labsl_flags_internal \
 	-labsl_flags_reflection \
